content/issues/2021-03-21-update-sites-not-updating.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Update sites not updating
 date: 2021-03-21 10:00:00
-resolved: false
+resolved: true
+resolvedWhen: 2021-03-21 12:42:00
```

Comment:
Today's the 22nd.

Reply:
Dang, today was not my today. Thanks

Reply:
Fixed, at least for the honor :p https://github.com/jenkins-infra/status/commit/be656061ea99fee5215db17244efd7bce1b67fd6